### PR TITLE
WIP - changes to templates to accomodate new styles

### PR DESCRIPTION
**Pull request by donjo**
These are the set of changes necessary to make things look the same as they did before after eregs/notice-and-comment#312 goes through.

### CODE REVIEW

File: regulations/static/regulations/css/less/module/comment.less [hunk 1]
```diff
@@ -451,6 +452,7 @@ Write Mode
 
 .comment-header {
   margin: 0.6em 0;
+  text-transform: none;
```
Comment (cmc333333) [inline regulations/static/regulations/css/less/module/comment.less]: Is this needed? It doesn't look like a text-transform is applied to this

File: regulations/static/regulations/css/less/module/comment.less [hunk 2]
```diff
@@ -333,8 +333,9 @@ Write Mode
   .comment-index-items {
     margin: 0 0 15px 0;
 
-    & > h3 {
+    & > h4 {
       margin: 0.6em 0 0 0;
+      text-transform: none;
```
Comment (cmc333333) [inline regulations/static/regulations/css/less/module/comment.less]: Looks like this rule specifically applies to the text "Your Comment", which doesn't have a text-transform we need to unset? Similar to the below.

Comment (cmc333333) [inline regulations/static/regulations/css/less/module/comment.less]: Oops; sorry, I had your PR in notice-and-comment applied locally.

**Comment by cmc333333** [comment]
Rather than _unsetting_ the transform set in `typography.less`, would it make sense to not set this transform there and only include it in regtext (for backwards compatibility?). Otherwise, this is good; is it ready for merging?

**Comment by donjo** [comment]
In the future I'd like to set transforms on the more specific elements rather than all h4s globally. However, the style you referenced ended up being redundant based since I had to put a `text-transform: none` on the h4 (which was necessary because of the preamble text and `text-transform: uppercase` can't be removed from the global style in -site for backwards compatibility reasons).

I'll remove that line and otherwise this should be ok.

**Comment by cmc333333** [comment]
:+1: We might be able to do something like
```css
.old-eregs-classname h4 {
    text-transform: uppercase;
}
```
but I agree, we can swing back over the summer.

**Pull request merged**